docs/3x3/Methods/CFOP.md

```diff
@@ -42,24 +43,24 @@ To view a transcribed table of the F2L algorithms click below.
 <details>
   <summary>Transcribed F2L Algorithms</summary>
 
-<table border="1">
+<table border="1" className={styles.table}>
     <tr>
-        <td rowspan="3"><strong>Corner</strong></td>
-        <td colspan="5" align="center"><strong>Edge (In First Two Layers)</strong></td>
+        <th rowspan="3">Corner</th>
+        <th colspan="5">Edge (In First Two Layers)</th>
     </tr>
-    <tr align="center">
-        <td><strong>FR</strong></td>
-        <td><strong>FL</strong></td>
-        <td><strong>BR</strong></td>
-        <td><strong>BL</strong></td>
-        <td><strong>FU</strong></td>
+    <tr>
+        <th>FR</th>
+        <th>FL</th>
+        <th>BR</th>
+        <th>BL</th>
+        <th>FU</th>
     </tr>
-    <tr align="center">
-        <td><strong>RF</strong></td>
-        <td><strong>LF</strong></td>
-        <td><strong>RB</strong></td>
-        <td><strong>LB</strong></td>
-        <td><strong>UF</strong></td>
+    <tr>
+        <th>RF</th>
+        <th>LF</th>
+        <th>RB</th>
+        <th>LB</th>
+        <th>UF</th>

```

Comment:
I changed these from HTML data cells to header cells, so they are bolded and centered by default